lib/erb_lint/linters/deprecated_classes.rb

```diff
@@ -8,17 +8,21 @@ class DeprecatedClasses < Linter
 
       def initialize(config)
         @deprecated_ruleset = []
-        config['rule_set'].each do |rule|
-          rule['deprecated'].each do |class_expr|
-            @deprecated_ruleset.push(
-              class_expr: class_expr,
-              suggestion: rule['suggestion']
-            )
+        unless config['rule_set'].nil?
+          config['rule_set'].each do |rule|
+            break if rule['deprecated'].nil?
```

Comment:
This should have been `next` but it will be removed regardless once I use `fetch` below.